lib/features/rules/BpmnRules.js

```diff
@@ -117,6 +117,22 @@ BpmnRules.prototype.init = function() {
     return canResize(shape, newBounds);
   });
 
+  this.addRule('elements.create', function(context) {
+    var elements = context.elements,
+        source = context.source,
+        target = context.target,
+        position = context.position;
+
+    return every(elements, function(element) {
+
+      if (isConnection(element)) {
+        return find(elements, element.source) && find(elements, element.target);
```

Comment:
What is the rationale behind this change? Why not `canConnect(element.source, element.target)`?

Reply:
Ooops. It looks like this was meant to be used to check wether I can copy a connection. Will fix. 🛠